tests/Load/utils/insertUsersUtils.js

```diff
@@ -0,0 +1,197 @@
+                'users': batch
+            });
+
+            const request = {
+                method: 'POST',
+                url: `${this.utils.getBaseHttpUrl()}/batch`,
+                body: payload,
+                params: this.utils.getJsonHeader(),
+            };
+
+            yield [request, userPasswords];
+        }
+    }
+
+    prepareRequestBatch(numberOfUsers, batchSize) {
+        const numberOfRequests = Math.ceil(numberOfUsers / batchSize);
+        const generator = this.requestGenerator(numberOfRequests, batchSize);
+        const requestBatch = [];
+        const userPasswords = {};
+
+        for (let requestIndex = 0; requestIndex < numberOfRequests; requestIndex++) {
+            const {value, done} = generator.next();
+            if (done) break;
+            const [request, passwords] = value;
+            requestBatch.push(request);
+            Object.assign(userPasswords, passwords);
+        }
+
+        return [requestBatch, userPasswords];
+    }
+
+    insertUsers(numberOfUsers) {
+        const batchSize = Math.min(this.config.batchSize, numberOfUsers);
+        const users = [];
+
+        const [requestBatch, userPasswords] = this.prepareRequestBatch(numberOfUsers, batchSize);
+
+        const responses = http.batch(requestBatch);
+
+        responses.forEach((response) => {
+                JSON.parse(response.body).forEach((user) => {
+                    user.password = userPasswords[user.email];
+                    users.push(user);
+                });
+            }
+        );
+
+        return users;
+    }
+
+    countRequestForRampingRate(
+        startRps,
+        targetRps,
+        duration
+    ) {
+        const acceleration = (targetRps - startRps) / duration;
+
+        return Math.round((startRps * duration + acceleration * duration * duration / 2));
+    }
+
+    prepareUsers() {
+        return this.insertUsers(this.countTotalRequest());
+    }
+
+    countTotalRequest() {
+        const requestsMap = {
+            'run_smoke': this.countSmokeRequest.bind(this),
+            'run_average': this.countAverageRequest.bind(this),
+            'run_stress': this.countStressRequest.bind(this),
+            'run_spike': this.countSpikeRequest.bind(this)
+        };
+
+        let totalRequests = 0;
+
+        for (const key in requestsMap) {
+            if (this.utils.getCLIVariable(key) !== 'false') {
+                totalRequests += requestsMap[key]();
+            }
+        }
+
+        return Math.round(totalRequests * this.additionalUsersRatio);
+    }
+
+    countSmokeRequest() {
+        return this.smokeConfig.rps * this.smokeConfig.duration;
+    }
+
+    countAverageRequest() {
+        return this.countDefaultRequests(this.averageConfig);
+    }
+
+    countStressRequest() {
+        return this.countDefaultRequests(this.stressConfig);
+    }
+
+    countDefaultRequests(config) {
+        const riseRequests =
+            this.countRequestForRampingRate(
```

Comment:
i dont understand why there is the next line, make this in one line, this makes the code a little bit harder to understand, and we need to fix this case everywhere

const riseRequests = this.countRequestForRampingRate